runtime/gc_glue_java/ArrayletObjectModel.hpp

```diff
@@ -33,6 +33,9 @@
 #include "j9consts.h"
 #include "modron.h"
 #include "modronopt.h"
+#include "ModronAssertions.h"
+#undef UT_MODULE_LOADED
+#undef UT_MODULE_UNLOADED
```

Comment:
Why you need this two lines?

Reply:
I tried building without those 2 lines, but I get an error saying how `UT_MODULE_LOADED` and 
`UT_MODULE_UNLOADED` are already defined.

This meant that in `ArrayletObjectModelBase.hpp`, `Bits.hpp`, `Math,hpp` or `SlotObject.hpp`, `UT_MODULE_LOADED` and `UT_MODULE_UNLOADED` are already defined.

Reply:
ok, remove all three lines and rely on proper includes in higher level

Reply:
Sounds like a bug in a header file, possibly a missing include gate.

Reply:
yes this is old problem with `#include "ut_j9mm.h"`